astra/src/main/java/com/slack/astra/chunk/ReadOnlyChunkImpl.java

```diff
@@ -583,7 +583,9 @@ public SearchResult<T> query(SearchQuery query) {
           searchEndTime,
           query.howMany,
           query.aggBuilder,
-          query.queryBuilder);
+          query.queryBuilder,
+          query.includeFields,
```

Comment:
I think making this a single `source` field probably makes more sense than individual parameters, so that it matches the original API closer. Especially as we'll be adding quite a few extra args here.